src/init.ts

```diff
@@ -15,36 +15,10 @@ import registerSettings, { FORIEN_UNIDENTIFIED_ITEMS_MODULE_NAME } from './modul
 // Import TypeScript modules
 import registerDerivedItemSheetClass from './modules/ItemSheet';
 import registerContextMenuHook from './modules/ContextMenu';
-import { checkSettingsInitialized, getGame } from './modules/settings';
 import Identification from './modules/Identification';
 import registerItemClassMethod from './modules/Item';
-
-export let debugEnabled = 0;
-// 0 = none, warnings = 1, debug = 2, all = 3
-export const debug = (...args) => {
-  if (debugEnabled > 1) console.log(`DEBUG:${FORIEN_UNIDENTIFIED_ITEMS_MODULE_NAME} | `, ...args);
-};
-export const log = function (...args) {
-  console.log(`forien-unidentified-items | `, ...args);
-};
-export const warn = (...args) => {
-  if (debugEnabled > 0) console.warn(`${FORIEN_UNIDENTIFIED_ITEMS_MODULE_NAME} | `, ...args);
-};
-export const error = (...args) => console.error(`${FORIEN_UNIDENTIFIED_ITEMS_MODULE_NAME} | `, ...args);
-export const timelog = (...args) => warn(`${FORIEN_UNIDENTIFIED_ITEMS_MODULE_NAME} | `, Date.now(), ...args);
-
-export const i18n = (key) => {
-  return getGame().i18n.localize(key);
-};
-export const i18nFormat = (key, data = {}) => {
-  return getGame().i18n.format(key, data);
-};
-
-export const setDebugLevel = (debugText: string) => {
-  debugEnabled = { none: 0, warn: 1, debug: 2, all: 3 }[debugText] || 0;
-  // 0 = none, warnings = 1, debug = 2, all = 3
-  if (debugEnabled >= 3) CONFIG.debug.hooks = true;
-};
+import CONSTANTS from './modules/constants';
```

Comment:
As a matter of "standard" among the modules in general I have seen that the file in question is always on `scripts / contants.js` or` module / contants.js` . For me it would be better to leave it here.
~`No problem on ym side for the `CONSTANTS as C ` suggestion.`~ Come to think of it, no it's better not to use `as` as people tend to" get confused "when reading the code